tools/check_format.py

```diff
@@ -875,6 +875,11 @@ def checkErrorMessages(error_messages):
   # error_messages.
   def ownedDirectories(error_messages):
     owned = []
+    maintainers = [
+        '@mattklein123', '@htuch', '@alyssawilk', '@zuercher', '@lizan', '@snowp', '@junr03',
```

Comment:
It would be nice if we could make this an independent Python file with just the dictionary that we import here. Then we can reference this Python list in other places/scripts.

Reply:
Mind if we do that when/if we want it in other places?  We haven't even needed this list until now.

Reply:
Sure, I don't think it's a huge deal to split it out, but up to you.